shimmingtoolbox/read_nii.py

```diff
@@ -67,11 +76,19 @@ def image_type(json_data):
         return img_type
 
 
-def convert_siemens_phase(image, info):
+def rescale_siemens_phase(image):
```

Comment:
Is it necessary to add another function to rescale the phase that contains only one line? If rescale is gonna be used in other funciton from other manufacturers, you should name it more globally. If it is only for siemens and used once, adding a function is unoptimal in this case and makes it more difficult to maintain.